jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/SideNav.java

```diff
@@ -34,34 +36,23 @@ public UIElement getSideNav(String position) {
         return element;
     }
 
-    @JDIAction("Get '{name}' side nav content")
-    public UIElement getEvents() {
-        return getContent().find(".example-events");
-    }
-
     @JDIAction("Get '{name}' side nav content")
     public WebList getSideNavLinks() {
-        return getSideNav().finds("a");
-    }
-
-    @JDIAction("Get '{name}' side nav content")
-    public WebList getResponsiveResults() {
-        return getContent().finds("p");
+        return this.finds("a");
     }
 
     @JDIAction("Get '{name}' side nav items")
     private WebList getSideNavItems() {
         return this.finds(".mat-sidenav");
```

Comment:
How it is possible to have sidenav inside sidenav?

Reply:
fixed